Frontend/src/app/views/dashboard/time/time.component.scss

```diff
@@ -138,13 +138,39 @@ button {
   width: 100%;
   border: unset !important;
 }
+
+// Undo styles to popover made by nav //
+::ng-deep .cdk-overlay-pane {
+  overflow: initial;
+  width: initial;
+  right: initial;
+}
+::ng-deep .nb-theme-default nb-popover {
+  width: initial;
+  background-color: initial;
+  padding: initial;
+  a {
+    font-size: initial;
+    font-weight: inital;
+    font-family: initial;
+    text-decoration: initial;
+    color:initial;
+  }
+}
+::ng-deep .nb-theme-default nb-popover.nb-overlay-bottom-start .arrow {
+  display: initial;
+}

```

Comment:
Something weird is going on with the popovers they are all pulled to the left side of the screen.

Reply:
Popover styling has been removed.